Applications/Spire/Include/Spire/Spire/AnyValueModel.hpp

```diff
@@ -101,6 +101,9 @@ namespace Spire {
   template<typename T>
   QValidator::State AnyValueModel::WrapperValueModel<T>::set(
       const Type& value) {
+    if(!value.has_value()) {
```

Comment:
Undo this change.

Reply:
The intention is to clear the value of the `AnyValueModel` from the `OpenFilterPanel`. The `OpenFilterPanel` knows that the type of the current of the `AnyInputBox` is `AnyRef`, and it can set an empty `AnyRef` to the `current` to clear the `AnyInputBox`. If this code gets removed, `any_cast` will return a invalid pointer because the type of the empty `AnyRef` is void but the casted type is `std::any` which is the type of the `current` of the `ComboBox`, and there is no way to clear the `AnyInputBox` from the `OpenFilterPanel`.